pyocd/coresight/cortex_m.py

```diff
@@ -798,12 +769,10 @@ def read_core_registers_raw(self, reg_list):
                 raise ValueError("unknown reg: %d" % reg)
             elif is_fpu_register(reg) and (not self.has_fpu):
```

Comment:
Shouldn't `is_fpu_register()` be updated to include double float regs?

Reply:
Hm, probably should have been that already. I think this would error correctly anyway as doubles recurse back in to do the single read, but will adjust.